classquiz/worker/storage.py

```diff
@@ -96,7 +76,21 @@ async def quiz_update(ctx, old_quiz: Quiz, quiz_id: uuid.UUID):
                 continue
             try:
                 await new_quiz.storageitems.remove(item)
-            except ormar.exceptions.NoMatch:
+            except ormar.exceptions.NoMatch as e:
+                logging.critical(e)
+                continue
+            change_made = True
+    for music in removed_musics:
+        if "--" in music:
```

Comment:
Not needed, as this ensures legacy compatibility

Reply:
Which part? The "--" check ?

Reply:
Exactly. I left the comment for that line, but GitHub doesn't tell you so. So yes, the part with the `--`.